docs/setting-up.md

```diff
@@ -30,6 +30,14 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
     upstream        https://github.com/TEAMMATES/teammates.git (fetch)
     upstream        https://github.com/TEAMMATES/teammates.git (push)
     ```
+1. Retrieve remote-tracking branches from the upstream repo.
```

Comment:
This works, but you can combine it with the previous step instead. That way there's no need for `git remote -v` anymore.